flygym/tests/test_odor.py

```diff
@@ -1,7 +1,10 @@
 import numpy as np
 
-from flygym import NeuroMechFly, Parameters
 from flygym.arena import OdorArena
+from flygym import Fly, Camera, SingleFlySimulation
+from flygym.examples.locomotion import HybridTurningController
```

Comment:
I renamed a bunch of stuff (see #154). Branching from `examples-refactoring` or `examples-tests` would give you the most up-to-date version.